include/unifex/let_value_with_stop_source.hpp

```diff
@@ -51,7 +51,7 @@ template<typename Operation, typename Receiver>
 class _stop_source_receiver<Operation, Receiver>::type {
 public:
     explicit type(Operation& op, Receiver&& r) :
-        op_(op), receiver_{std::forward<Receiver>(r)}
+        op_(op), receiver_{std::move(r)}
```

Comment:
This is better than it was, but this whole file looks like it needs to have the value categories of the various forwarded arguments reconsidered.